attributes/default.rb

```diff
@@ -15,7 +15,7 @@
 
 default['jellyfish']['user'] = 'jellyfish'
 default['jellyfish']['group'] = 'jellyfish'
-default['jellyfish']['ruby_version'] = '2.2.0'
+default['jellyfish']['ruby_version'] = '2.2.1'
```

Comment:
Yeah, Unfortunately it is. The ruby_version variable is not accessible from the attributes file. It looks like attributes are sourced first. I am using the default['jellyfish']['ruby_version'] to construct other variables in the attributes file. For example default['rbenv']['gem_exec']. I can't construct that variable with out default['jellyfish']['ruby_version'] in the attributes file.